static_frame/core/index_hierarchy.py

```diff
@@ -2165,12 +2185,14 @@ def __contains__(self: IH, # type: ignore
 
     def unique(self: IH,
             depth_level: DepthLevelSpecifier = 0,
+            order_by_occurrence: bool = False,
```

Comment:
Same here: default to True?

Reply:
Same response here: https://github.com/InvestmentSystems/static-frame/pull/578#discussion_r971392814

Reply:
I see your point. I was thinking that, for backwards compatibility with what `Index` is already doing, `True` was necessary; i.e., since `Index` was already giving the values in `order_of_occurrence`, the default there and for `IndexHierarchy` should be `True`. I know that `Index` will return the same array regardless of the value of that parameter. But I guess before these changes, `IndexHierarchy` was not returning the values in order of occurrence, right?

Reply:
Correct. `IndexHierarchy.unique` was _not_ returning the unique values in occurrence order (unless the data was an unsortable object numpy array).

Technically,, `index.unique(order_of_occurrence=True)` == `index.unique(order_of_occurrence=False)`, since all `.unique(...)` does is return `.values`, so leaving that default to `False` will not change anything. I interpret the `order_of_occurrence=False` to mean, "We do not guarantee the values will be ordered by how they appear." It doesn't mean, "They definitely won't be ordered by how they appear."

Reply:
Ok, agreed; lets keep as you have it!